modules/uncertainty/R/run.ensemble.analysis.R

```diff
@@ -131,8 +131,9 @@ run.ensemble.analysis <- function(settings,plot.timeseries=NA, ensemble.id=NULL,
 ##'
 ##' @author Michael Dietze, Ryan Kelly
 ##'
-read.ensemble.ts <- function(model, ensemble.id=NULL, variable=NULL, start.year=NULL, end.year=NULL) {
+read.ensemble.ts <- function(settings, ensemble.id=NULL, variable=NULL, start.year=NULL, end.year=NULL) {
```

Comment:
@mdietze the only function that calls read.ensemble.ts besides the workflow is sda.particle, I haven't modified it